/**:
  ros__parameters:
    # Total planning attempts
    max_solutions: 1

    # execute the pp
    execute: true

    # Planning group and link names
    arm_group_name: "arm_ur5e"
    eef_name: "gripper_robotiq"
    hand_group_name: "gripper_robotiq"
    hand_frame: "gripper_robotiq_arg2f_base_link"

    # Poses
    hand_open_pose: "gripper_open"
    hand_close_pose: "gripper_close"
    arm_home_pose: "pick_handover"

    # Scene frames
    world_frame: "base_link"
    table_reference_frame: "base_link"
    object_reference_frame: "base_link"
    surface_link: "table"

    # Collision object for picking
    # CYLINDER object specifications
    object_name: "object"
    object_dimensions: [0.06, 0.06,0.06] # [height, radius]
    object_pose: [1.22, 0.12, 0.9, 0.0, 0.0, 0.0]

    # Table model
    spawn_table: true
    table_name: "table"
    table_dimensions: [0.4, 0.5, 0.1]  # [length, width, height]
    table_pose: [1.38, 0.0, 0.9, 0.0, 0.0, 0.0]

    # Gripper grasp frame transform [x,y,z,r,p,y]
    grasp_frame_transform: [0.0, 0.0, 0.2, 0.0, 1.57, 0.0]
    
    #[0.0, 0.0, 0.2, 0.0, 3.14, 0.0]

    # Place pose [x,y,z,r,p,y]
    place_pose: [0.6, 0.4, 0.0, 0.0, 0.0, 0.0]
    place_surface_offset: 0.0001 # place offset from table

    # Valid distance range when approaching an object for picking
    approach_object_min_dist: 0.1
    approach_object_max_dist: 0.15

    # Valid height range when lifting an object after pick
    lift_object_min_dist: 0.01
    lift_object_max_dist: 0.1
